third_party/build/patches/libfluid/libfluid_base_patches/EVLOOP_NO_EXIT_ON_EMPTY_compat.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/fluid/base/EventLoop.cc b/fluid/base/EventLoop.cc
```

Comment:
Where does that patch come from stackoverflow? Is that the only way to get it? (maybe cherry-pick if a commit has been created)

Reply:
no, patch was my own work (originally for CentOS 8 experimental build) -- would be great if we could get libfluid to import it